src/main/java/stsjorbsmod/cards/cull/Withering.java

```diff
@@ -0,0 +1,55 @@
+package stsjorbsmod.cards.cull;
+
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.actions.DamageWithOnKillEffectAction;
+import stsjorbsmod.actions.PermanentlyModifyDamageAction;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.cards.OnDrawCardSubscriber;
+import stsjorbsmod.characters.Cull;
+import stsjorbsmod.patches.EphemeralField;
+import stsjorbsmod.powers.WitherPower;
+
+import static stsjorbsmod.JorbsMod.JorbsCardTags.LEGENDARY;
+
+public class Withering extends CustomJorbsModCard implements OnDrawCardSubscriber {
+    public static final String ID = JorbsMod.makeID(Withering.class);
+
+    private static final AbstractCard.CardRarity RARITY = AbstractCard.CardRarity.BASIC;
+    private static final AbstractCard.CardTarget TARGET = AbstractCard.CardTarget.ENEMY;
+    private static final AbstractCard.CardType TYPE = CardType.CURSE;
+    public static final AbstractCard.CardColor COLOR = CardColor.CURSE;
+
+
+    private static final int COST = -2;
```

Comment:
The design sheet says the intention is for it to be playable at cost 1 (to gain 1 intangible)